Distributed Information Systems/about.html

```diff
@@ -81,9 +81,48 @@
                     <h5>OUR COMPANY</h5>
                     <h2>Some Fine<br>Words About Us</h2>
                     <!-- Replace About Us Text Under -->
-                    <p>Vestibulum ac diam sit amet quam vehicula elementum amet est on dui. Nulla porttitor accumsan
-                        tincidunt.Vestibulum ac diam sit amet.Quam vehicula elementum amet est on dui. Nulla porttitor
-                        accumsan tincidunt.</p>
+                    <p>Founded in 2022, DIS Services is a leading Technology Consulting and Management company with a
+                        top-calibre team of consultants.
+                        We pride ourselves on our client-centric approach that has proven successful since the company&apos;s
+                        inception.
+                        We assist our clients to stay ahead in the ever-changing technological landscape by assisting them in
+                        bringing their products to the market.
+                        Our mission is to amplify our client&apos;s products in the technology market and save companies money
+                        in the process.
+        
+                    </p>
+                    <div>
+                        <div style="float: left; width: 50%"><img alt="Shailendra Singh" src=".\images\shai" width="300" height="400"><br><strong>Shailendra Singh</strong><br><p>Shailendra Singh is a Masters Student at the University of South Florida.<br>
```

Comment:
Since most web servers are Unix based when selecting the source, use forward slash /, instead of backward slash. Also, since the IMG extensions are not there, the images are not showing in the preview.

try ./images/shail.jpg